openy_gated_content.post_update.php

```diff
@@ -126,3 +126,45 @@ function openy_gated_content_post_update_eventseries_livestream(&$sandbox) {
 function openy_gated_content_post_update_eventseries_meeting(&$sandbox) {
   _openy_gated_content_permissions($sandbox, 'eventseries', 'virtual_meeting', 'id');
 }
+
+/**
+ * Set gated_content paragraphs title and description if it was empty.
+ */
+function openy_gated_content_post_update_paragraph_headline(&$sandbox) {
+  $prgf_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $gated_content_prgf = $prgf_storage->loadByProperties(['type' => 'gated_content']);
```

Comment:
this will eat a lot of memory when there is a lot of content
could we load by chunks by 50 instead of all?

Reply:
@podarok In most Y's cases, I think there will be only one gated_content paragraph per site, so no need to limit this query.

cc @retif